cypress/utils/data_utils.ts

```diff
@@ -135,6 +134,21 @@ export function getRandomCredentialsData(
             password: password,
         };
     }
+
+    if (type === CredentialType.jira) {
+        if (useRealUser) {
+            email = Cypress.env("jira_email");
```

Comment:
These fields need to be added in cypress.json as well .
Pls add the test account creds.

Reply:
I don't wanna put here data that allows access to somewhere, it is sensitive, no? Not secure. It will be passed as an additional parameter in Jenkins